app/server/tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
     "compilerOptions": {
-        "target": "es2019",
+        "target": "esnext",
+        "module": "commonjs",
         "esModuleInterop": true,
         "resolveJsonModule": true,
         "outDir": "dist",
-        "moduleResolution": "Node"
+        "moduleResolution": "node"
```

Comment:
This combination of values allows both `import`s and `require`s in the server code. I'm not sure why this had stopped working in the container but not on metal... There are also options for `nodenext` which should work too, but get compilation complaints about instability and demands to use TS nightly builds..